connexion/http_facts.py

```diff
@@ -2,3 +2,14 @@
     'application/x-www-form-urlencoded',
     'multipart/form-data'
 ]
+
+METHODS = set([
+    "get",
+    "put"
```

Comment:
It turns out the issue is a missing comma here.

Reply:
damn... sorry for that aha

I will add a check in my test, this evening, to see if a route has been created in flask app for each method, before merging